src/main/views/modalView.ts

```diff
@@ -48,13 +49,17 @@ export class ModalView<T, T2> {
         try {
             this.view.webContents.loadURL(this.html);
         } catch (e) {
-            log.error('there was an error loading the modal:');
-            log.error(e);
+            this.log.error('there was an error loading the modal:');
+            this.log.error(e);
         }
 
         this.contextMenu = new ContextMenu({}, this.view);
     }
 
+    private get log() {
+        return logger.withPrefix('ModalView', this.key);
```

Comment:
Are we expecting the key to change? If not, we may want to assign this on the constructor, so we don't have to do all the `appendToLog` logic every time we need to log something. 0/5

Reply:
Yeah you're right, it's in the constructor now.